src/gui_common/CustomRichTextLabel.cs

```diff
@@ -446,7 +446,9 @@ string GetResizedImage(string imagePath, int width, int height, int ascent)
     /// </summary>
     private void ParseCustomTags()
     {
-        if (extendedBbcode == null)
+        var translated = TranslationServer.Translate(extendedBbcode);
```

Comment:
What if extendedBbcode is null? Would it make sense to check for that explicitly and skip the call to Translate?

Reply:
I think so.